LICENSE

```diff
@@ -1,121 +0,0 @@
-CC0 1.0 Universal
```

Comment:
we don't want to delete this license file.  We can't change the license.  But add a Columbia copyright statement for 2018-2024